### DISCUSSION TITLE
Write macro placement position to a tcl file

### DISCUSSION THREAD

**Discussion by vijayank88** [The-OpenROAD-Project/OpenROAD#5041] [General]
Similar to `write_pin_placement` option, is there a option to write macro placement position while using automatic macro placement.
So that macro position will be re-used for some specific requirement with manual macro placement again.

**Comment by maliberty** [comment]
This is more of an issue than a discussion

**Comment by maliberty** [comment]
https://github.com/The-OpenROAD-Project/OpenROAD/issues/5043

**Comment by AcKoucher** [comment]
@vijayank88 FYI, there's the `write_macro_placement` flag in mpl2. However it's currently deprecated due to a new OR command for general macro placement writing that was implemented in #4557.

I've opened #5046 to update the flag so that mpl2 will write the entire macro placement instead of only those macros that it placed.
